cpp/ql/test/library-tests/ir/range-analysis/SimpleRangeAnalysis_tests.cpp

```diff
@@ -888,7 +888,7 @@ void notequal_variations(short n, float f) {
   }
 
   if (n >= 5) {
-    if (2 * n - 10 == 0) { // $ overflow=+
+    if (2 * n - 10 == 0) { // no overflow
```

Comment:
Why does this change?

Reply:
`n` is a short, and `2` is a int. So `n` gets promoted to an int, and we infer that `n <= SHRT_MAX` which fits well within an integer (which is the resulting type of `2 * n`)